inc/abstractrightsdropdown.class.php

```diff
@@ -79,6 +79,7 @@ public static function show(string $name, array $values): string {
       $params = [
          'values' => $values,
          'valuesnames' => self::getValueNames($values),
+         'multiple'    => true,
```

Comment:
I think this is the only JS Ajax Dropdown with multiple selection. Confirmation needed.